### ISSUE TITLE
What kind of bugs we can reports

### ISSUE THREAD

**Issue by beshoo** [ArchiDroid/android_device_samsung_i9300#152]
Dear @JustArchi. 
Thank you for your time, replying all posts fast. 
No doubt you are a wonderful dev. 

And we, web developer,  want to offer you a correct help, bugs report, as i do respect your time,  i need to know what kind of bugs i can report it to or to CM team. 

Some time i faced battery problems,  eg 10% lost each hour while screen is off and all background app greenified. By greenify app. 

Is this bug is yours or CM Team. 

Please advice and what the documentation we have to attach other than logcat or last_cm. 

Regards.

**Comment by JustArchi** [comment]
In general I accept all bugs that might be connected with device tree, which is - a bug that is specific to i9300 and/or other exynos 4 devices and is not caused by CM or Android framework.

However, as I'm giving up on i9300, and the next build is supposed to be the last one, I don't actually accept neither fix anything, as I just long my last willings to work on this crap.

**Comment by frazzmark** [comment]
I don't think that you should be so pessimist. Leaving bugs open could point to others the problems to solve.

**Comment by JustArchi** [comment]
Then go ahead and solve them, because during whole this time nobody apart from Moster and me took any bug into consideration.

**Comment by frazzmark** [comment]
Well, I didn't want to start some big argument. I just think that, since there are people interested in this problem, when you decide to leave the development (which of course you've got every right to do) leaving the issues open should point the things to improve to all the others.
Look at it this way: maybe nobody except moster did any commit because some issues were closed prematurely.
Anyway, this is your project. Feel free to do as you'd like.

**Comment by JustArchi** [comment]
I'm not going to close any bug that is device-based and should be fixed, I just marked them as won't fix, because obviously I won't be fixing them.

I don't know what are you refering to.

**Comment by TwoD** [comment]
Thanks for this clarification, JustArchi! I've seen you answer similar questions before, but even if these answers are tedious to repeat, it really makes them easier to find for others.

All the time you have spent working on this has been much appreciated and it saved at least me from having to buy a new phone - because of lack of OS updates - when I really did not have the cash to spend on it. I'm not familiar enough with building/hacking Android to do it myself so you've been a great help.
